packages/bot-web-ui/src/app.jsx

```diff
@@ -0,0 +1,129 @@
+import { 
+    DBot,
+    ServerTime }            from '@deriv/bot-engine';
+import { reaction }         from 'mobx';
+import { Provider }         from 'mobx-react';
+import React                from 'react';
+import                           './public-path'; // Leave this here! OK boss!
+import FooterExtension      from './components/footer-extension.jsx';
+import MainContent          from './components/main-content.jsx';
+import NotificationMessages from './components/notification-messages.jsx';
+import QuickStrategy        from './components/quick-strategy.jsx';
+import RunPanel             from './components/run-panel.jsx';
+import Toolbar              from './components/toolbar.jsx';
+import RootStore            from './stores';
+import GTM                  from './utils/gtm';
+import                      './assets/sass/app.scss';
+
+class App extends React.Component {
+    constructor(props) {
+        super(props);
+        const { passthrough: { WS, root_store } } = props;
+        this.root_store = new RootStore(root_store, WS, DBot);
+
+        GTM.init(this.root_store);
+        ServerTime.init(root_store.common);
+        
+        const { flyout, toolbar, quick_strategy } = this.root_store;
+        // dont pass the rootstore in, if you need a prop define it in dbot-engine-store ans pass it through
```

Comment:
What's the argument against passing the entire `root_store`? I think it'll make it easier to understand to which data you have access to as the structure will be exactly the same.